### ISSUE TITLE
extremely long loading time for notebook

### ISSUE THREAD

**Issue by linuxrider** [flathub/app.rednotebook.RedNotebook#1]
The loading time of the notebook files is extremly long which is not the case for the native app.
Following are the logs for native and flatpaked version.
(native version is 2.10 and flatpaked 2.8, but also native 2.8 was starting up quickly)
```
~$ rednotebook
Adding /usr/share/rednotebook to sys.path
2019-03-25 08:00:37,594 INFO     Writing log to file "<userhome>/.rednotebook/rednotebook.log"
2019-03-25 08:00:37,594 INFO     System encoding: utf-8
2019-03-25 08:00:37,594 INFO     Language code: de_DE
2019-03-25 08:00:37,834 INFO     Running in portable mode: False
2019-03-25 08:00:37,834 INFO     First Start: False
2019-03-25 08:00:37,834 INFO     RedNotebook version: 2.10
2019-03-25 08:00:37,842 INFO     System info: machine: x86_64, platform: Linux-4.18.0-16-generic-x86_64-with-Ubuntu-18.10-cosmic, processor: x86_64, python_version: 3.6.7, release: 4.18.0-16-generic, system: Linux, GTK: (3, 24, 4), Glib: (2, 58, 0), PyGObject: (3, 30, 1), YAML: 3.12

(rednotebook:12824): Gtk-WARNING **: 08:00:38.084: Content added to the action area of a dialog using header bars

(rednotebook:12824): Gtk-WARNING **: 08:00:38.085: Content added to the action area of a dialog using header bars

(rednotebook:12824): Gtk-WARNING **: 08:00:38.179: Content added to the action area of a dialog using header bars

(rednotebook:12824): Gtk-WARNING **: 08:00:38.179: Content added to the action area of a dialog using header bars

(rednotebook:12824): Gtk-WARNING **: 08:00:38.253: Content added to the action area of a dialog using header bars

(rednotebook:12824): Gtk-WARNING **: 08:00:38.254: Content added to the action area of a dialog using header bars

(rednotebook:12824): Gtk-WARNING **: 08:00:38.344: Content added to the action area of a dialog using header bars

(rednotebook:12824): Gtk-WARNING **: 08:00:38.344: Content added to the action area of a dialog using header bars
2019-03-25 08:00:38,370 DEBUG    Default font: Ubuntu 11
2019-03-25 08:00:38,370 DEBUG    Default size: 11.0
2019-03-25 08:00:38,569 INFO     Cloud ignore list: ['#tags', 'and', 'comma', 'filter', 'separated', 'these', 'words']
2019-03-25 08:00:38,570 INFO     Cloud include list: ['job', 'juli', 'mtv', 'spam']
2019-03-25 08:00:38,570 DEBUG    Start compiling regexes
2019-03-25 08:00:38,571 DEBUG    Finished
2019-03-25 08:00:38,596 DEBUG    Tray icon visible: False
2019-03-25 08:00:38,597 INFO     Opening journal at '<journal folder>/'
2019-03-25 08:00:38,598 DEBUG    Starting to load files in dir "<journal folder>/"
2019-03-25 08:00:38,598 DEBUG    Loading file "<journal folder>/2012-01.txt"
2019-03-25 08:00:38,600 DEBUG    Loading file "<journal folder>/2012-02.txt"
2019-03-25 08:00:38,603 DEBUG    Loading file "<journal folder>/2012-03.txt"
2019-03-25 08:00:38,604 DEBUG    Loading file "<journal folder>/2012-04.txt"
2019-03-25 08:00:38,606 DEBUG    Loading file "<journal folder>/2012-05.txt"
2019-03-25 08:00:38,607 DEBUG    Loading file "<journal folder>/2012-06.txt"
2019-03-25 08:00:38,608 DEBUG    Loading file "<journal folder>/2012-07.txt"
2019-03-25 08:00:38,608 DEBUG    Loading file "<journal folder>/2012-08.txt"
2019-03-25 08:00:38,610 DEBUG    Loading file "<journal folder>/2012-09.txt"
2019-03-25 08:00:38,611 DEBUG    Loading file "<journal folder>/2012-10.txt"
2019-03-25 08:00:38,613 DEBUG    Loading file "<journal folder>/2012-11.txt"
2019-03-25 08:00:38,615 DEBUG    Loading file "<journal folder>/2013-02.txt"
2019-03-25 08:00:38,617 DEBUG    Loading file "<journal folder>/2013-03.txt"
2019-03-25 08:00:38,624 DEBUG    Loading file "<journal folder>/2013-04.txt"
2019-03-25 08:00:38,629 DEBUG    Loading file "<journal folder>/2013-05.txt"
2019-03-25 08:00:38,630 DEBUG    Loading file "<journal folder>/2013-06.txt"
2019-03-25 08:00:38,632 DEBUG    Loading file "<journal folder>/2013-07.txt"
2019-03-25 08:00:38,634 DEBUG    Loading file "<journal folder>/2013-08.txt"
2019-03-25 08:00:38,637 DEBUG    Loading file "<journal folder>/2013-09.txt"
2019-03-25 08:00:38,639 DEBUG    Loading file "<journal folder>/2013-10.txt"
2019-03-25 08:00:38,639 DEBUG    Loading file "<journal folder>/2013-11.txt"
2019-03-25 08:00:38,640 DEBUG    Loading file "<journal folder>/2013-12.txt"
2019-03-25 08:00:38,641 DEBUG    Loading file "<journal folder>/2014-01.txt"
2019-03-25 08:00:38,643 DEBUG    Loading file "<journal folder>/2014-02.txt"
2019-03-25 08:00:38,649 DEBUG    Loading file "<journal folder>/2014-03.txt"
2019-03-25 08:00:38,651 DEBUG    Loading file "<journal folder>/2014-04.txt"
2019-03-25 08:00:38,652 DEBUG    Loading file "<journal folder>/2014-05.txt"
2019-03-25 08:00:38,654 DEBUG    Loading file "<journal folder>/2014-06.txt"
2019-03-25 08:00:38,654 DEBUG    Loading file "<journal folder>/2014-07.txt"
2019-03-25 08:00:38,656 DEBUG    Loading file "<journal folder>/2014-08.txt"
2019-03-25 08:00:38,657 DEBUG    Loading file "<journal folder>/2014-09.txt"
2019-03-25 08:00:38,661 DEBUG    Loading file "<journal folder>/2014-10.txt"
2019-03-25 08:00:38,664 DEBUG    Loading file "<journal folder>/2014-11.txt"
2019-03-25 08:00:38,665 DEBUG    Loading file "<journal folder>/2014-12.txt"
2019-03-25 08:00:38,666 DEBUG    Loading file "<journal folder>/2015-01.txt"
2019-03-25 08:00:38,667 DEBUG    Loading file "<journal folder>/2015-02.txt"
2019-03-25 08:00:38,672 DEBUG    Loading file "<journal folder>/2015-03.txt"
2019-03-25 08:00:38,674 DEBUG    Loading file "<journal folder>/2015-04.txt"
2019-03-25 08:00:38,680 DEBUG    Loading file "<journal folder>/2015-05.txt"
2019-03-25 08:00:38,683 DEBUG    Loading file "<journal folder>/2015-06.txt"
2019-03-25 08:00:38,685 DEBUG    Loading file "<journal folder>/2015-07.txt"
2019-03-25 08:00:38,687 DEBUG    Loading file "<journal folder>/2015-08.txt"
2019-03-25 08:00:38,690 DEBUG    Loading file "<journal folder>/2015-09.txt"
2019-03-25 08:00:38,694 DEBUG    Loading file "<journal folder>/2015-10.txt"
2019-03-25 08:00:38,698 DEBUG    Loading file "<journal folder>/2015-11.txt"
2019-03-25 08:00:38,701 DEBUG    Loading file "<journal folder>/2015-12.txt"
2019-03-25 08:00:38,710 DEBUG    Loading file "<journal folder>/2016-01.txt"
2019-03-25 08:00:38,711 DEBUG    Loading file "<journal folder>/2016-02.txt"
2019-03-25 08:00:38,712 DEBUG    Loading file "<journal folder>/2016-03.txt"
2019-03-25 08:00:38,714 DEBUG    Loading file "<journal folder>/2016-04.txt"
2019-03-25 08:00:38,726 DEBUG    Loading file "<journal folder>/2016-05.txt"
2019-03-25 08:00:38,730 DEBUG    Loading file "<journal folder>/2016-06.txt"
2019-03-25 08:00:38,741 DEBUG    Loading file "<journal folder>/2016-07.txt"
2019-03-25 08:00:38,748 DEBUG    Loading file "<journal folder>/2016-08.txt"
2019-03-25 08:00:38,753 DEBUG    Loading file "<journal folder>/2016-09.txt"
2019-03-25 08:00:38,756 DEBUG    Loading file "<journal folder>/2016-10.txt"
2019-03-25 08:00:38,761 DEBUG    Loading file "<journal folder>/2016-11.txt"
2019-03-25 08:00:38,770 DEBUG    Loading file "<journal folder>/2016-12.txt"
2019-03-25 08:00:38,778 DEBUG    Loading file "<journal folder>/2017-01.txt"
2019-03-25 08:00:38,780 DEBUG    Loading file "<journal folder>/2017-02.txt"
2019-03-25 08:00:38,782 DEBUG    Loading file "<journal folder>/2017-03.txt"
2019-03-25 08:00:38,787 DEBUG    Loading file "<journal folder>/2017-04.txt"
2019-03-25 08:00:38,789 DEBUG    Loading file "<journal folder>/2017-05.txt"
2019-03-25 08:00:38,794 DEBUG    Loading file "<journal folder>/2017-06.txt"
2019-03-25 08:00:38,797 DEBUG    Loading file "<journal folder>/2017-07.txt"
2019-03-25 08:00:38,801 DEBUG    Loading file "<journal folder>/2017-08.txt"
2019-03-25 08:00:38,805 DEBUG    Loading file "<journal folder>/2017-09.txt"
2019-03-25 08:00:38,812 DEBUG    Loading file "<journal folder>/2017-10.txt"
2019-03-25 08:00:38,816 DEBUG    Loading file "<journal folder>/2017-11.txt"
2019-03-25 08:00:38,818 DEBUG    Loading file "<journal folder>/2017-12.txt"
2019-03-25 08:00:38,818 DEBUG    Loading file "<journal folder>/2018-01.txt"
2019-03-25 08:00:38,824 DEBUG    Loading file "<journal folder>/2018-03.txt"
2019-03-25 08:00:38,825 DEBUG    Loading file "<journal folder>/2018-04.txt"
2019-03-25 08:00:38,829 DEBUG    Loading file "<journal folder>/2018-05.txt"
2019-03-25 08:00:38,830 DEBUG    Loading file "<journal folder>/2018-06.txt"
2019-03-25 08:00:38,831 DEBUG    Loading file "<journal folder>/2018-07.txt"
2019-03-25 08:00:38,832 DEBUG    Loading file "<journal folder>/2018-08.txt"
2019-03-25 08:00:38,834 DEBUG    Loading file "<journal folder>/2018-09.txt"
2019-03-25 08:00:38,836 DEBUG    Loading file "<journal folder>/2018-10.txt"
2019-03-25 08:00:38,839 DEBUG    Loading file "<journal folder>/2018-11.txt"
2019-03-25 08:00:38,841 DEBUG    Loading file "<journal folder>/2018-12.txt"
2019-03-25 08:00:38,843 DEBUG    Loading file "<journal folder>/2019-01.txt"
2019-03-25 08:00:38,878 DEBUG    Loading file "<journal folder>/2019-02.txt"
2019-03-25 08:00:38,883 DEBUG    Loading file "<journal folder>/2019-03.txt"
2019-03-25 08:00:38,884 DEBUG    RedNotebook-Backup-2015-12-17.zip is not a valid month filename
2019-03-25 08:00:38,884 DEBUG    RedNotebook-Backup-Notebook-2016-01-06.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-Notebook-2016-01-17.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-Notebook-2016-02-07.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-Notebook-2016-02-29.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-Notebook-2016-03-31.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-05-01.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-06-08.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-07-03.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-08-02.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-08-03.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-08-09.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-09-02.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-10-07.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-11-08.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2016-12-05.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2017-02-14.zip is not a valid month filename
2019-03-25 08:00:38,885 DEBUG    RedNotebook-Backup-labjournal-2017-04-17.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2017-05-21.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2017-07-18.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2017-09-20.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2018-01-10.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2018-04-23.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2018-06-10.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2018-07-12.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2018-10-22.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2019-01-19.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2019-01-29.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2019-02-12.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    RedNotebook-Backup-labjournal-2019-03-03.zip is not a valid month filename
2019-03-25 08:00:38,886 DEBUG    Finished loading files in dir "<journal folder>/"
2019-03-25 08:00:39,750 INFO     Connected Signals: [<Signals.SIGHUP: 1>, <Signals.SIGINT: 2>, <Signals.SIGQUIT: 3>, <Signals.SIGABRT: 6>, <Signals.SIGTERM: 15>, <Signals.SIGTSTP: 20>]
2019-03-25 08:00:39,750 DEBUG    Start took 1.9162876605987549 seconds
2019-03-25 08:00:39,750 DEBUG    Trying to enter the gtk main loop
2019-03-25 08:00:39,893 DEBUG    Update the cloud
2019-03-25 08:00:41,521 DEBUG    Cloud updated
2019-03-25 08:00:41,525 INFO     Last backup was made 41 days ago
2019-03-25 08:00:41,525 WARNING  Last backup is older than 30 days.
2019-03-25 08:01:16,563 DEBUG    Main frame destroyed
2019-03-25 08:01:16,564 INFO     Nichts zu speichern
2019-03-25 08:01:16,564 INFO     Configuration has been saved to /home/jo/.rednotebook/configuration.cfg
2019-03-25 08:01:16,565 INFO     Goodbye!
2019-03-25 08:01:16,576 INFO     Peak memory: 117863808 KiB
```

```
[📦 app.rednotebook.RedNotebook ~]$ rednotebook
Adding /app/lib/python3.7/site-packages to sys.path
2019-03-25 07:58:30,438 INFO     Writing log to file "<userhome>/.rednotebook/rednotebook.log"
2019-03-25 07:58:30,438 INFO     System encoding: utf-8
2019-03-25 07:58:30,438 INFO     Language code: de_DE
2019-03-25 07:58:30,547 INFO     Using pyyaml for loading and dumping
2019-03-25 07:58:30,756 INFO     Running in portable mode: False
2019-03-25 07:58:30,756 INFO     First Start: False
2019-03-25 07:58:30,757 INFO     RedNotebook version: 2.8
2019-03-25 07:58:30,775 INFO     System info: machine: x86_64, platform: Linux-4.18.0-16-generic-x86_64-with-glibc2.2.5, processor: , python_version: 3.7.2, release: 4.18.0-16-generic, system: Linux, GTK: (3, 24, 5), Glib: (2, 58, 3), PyGObject: (3, 30, 4), YAML: 3.13

(rednotebook:3): Gtk-WARNING **: 07:58:30.937: Content added to the action area of a dialog using header bars

(rednotebook:3): Gtk-WARNING **: 07:58:30.937: Content added to the action area of a dialog using header bars

(rednotebook:3): Gtk-WARNING **: 07:58:30.990: Content added to the action area of a dialog using header bars

(rednotebook:3): Gtk-WARNING **: 07:58:30.991: Content added to the action area of a dialog using header bars

(rednotebook:3): Gtk-WARNING **: 07:58:31.036: Content added to the action area of a dialog using header bars

(rednotebook:3): Gtk-WARNING **: 07:58:31.036: Content added to the action area of a dialog using header bars

(rednotebook:3): Gtk-WARNING **: 07:58:31.101: Content added to the action area of a dialog using header bars

(rednotebook:3): Gtk-WARNING **: 07:58:31.101: Content added to the action area of a dialog using header bars
2019-03-25 07:58:31,129 DEBUG    Default font: Ubuntu 11
2019-03-25 07:58:31,129 DEBUG    Default size: 11.0
2019-03-25 07:58:31,267 INFO     Cloud ignore list: ['#tags', 'and', 'comma', 'filter', 'separated', 'these', 'words']
2019-03-25 07:58:31,267 INFO     Cloud include list: ['job', 'juli', 'mtv', 'spam']
2019-03-25 07:58:31,267 DEBUG    Start compiling regexes
2019-03-25 07:58:31,268 DEBUG    Finished
2019-03-25 07:58:31,290 DEBUG    Tray icon visible: False
2019-03-25 07:58:31,292 INFO     Opening journal at '<journal folder>/'
2019-03-25 07:58:31,292 DEBUG    Starting to load files in dir "<journal folder>/"
2019-03-25 07:58:31,293 DEBUG    Loading file "<journal folder>/2012-01.txt"
2019-03-25 07:58:31,334 DEBUG    Loading file "<journal folder>/2012-02.txt"
2019-03-25 07:58:31,487 DEBUG    Loading file "<journal folder>/2012-03.txt"
2019-03-25 07:58:31,582 DEBUG    Loading file "<journal folder>/2012-04.txt"
2019-03-25 07:58:31,697 DEBUG    Loading file "<journal folder>/2012-05.txt"
2019-03-25 07:58:31,752 DEBUG    Loading file "<journal folder>/2012-06.txt"
2019-03-25 07:58:31,789 DEBUG    Loading file "<journal folder>/2012-07.txt"
2019-03-25 07:58:31,795 DEBUG    Loading file "<journal folder>/2012-08.txt"
2019-03-25 07:58:31,872 DEBUG    Loading file "<journal folder>/2012-09.txt"
2019-03-25 07:58:31,947 DEBUG    Loading file "<journal folder>/2012-10.txt"
2019-03-25 07:58:32,078 DEBUG    Loading file "<journal folder>/2012-11.txt"
2019-03-25 07:58:32,363 DEBUG    Loading file "<journal folder>/2013-02.txt"
2019-03-25 07:58:32,490 DEBUG    Loading file "<journal folder>/2013-03.txt"
2019-03-25 07:58:33,303 DEBUG    Loading file "<journal folder>/2013-04.txt"
2019-03-25 07:58:33,694 DEBUG    Loading file "<journal folder>/2013-05.txt"
2019-03-25 07:58:33,789 DEBUG    Loading file "<journal folder>/2013-06.txt"
2019-03-25 07:58:33,880 DEBUG    Loading file "<journal folder>/2013-07.txt"
2019-03-25 07:58:34,017 DEBUG    Loading file "<journal folder>/2013-08.txt"
2019-03-25 07:58:34,227 DEBUG    Loading file "<journal folder>/2013-09.txt"
2019-03-25 07:58:34,387 DEBUG    Loading file "<journal folder>/2013-10.txt"
2019-03-25 07:58:34,391 DEBUG    Loading file "<journal folder>/2013-11.txt"
2019-03-25 07:58:34,425 DEBUG    Loading file "<journal folder>/2013-12.txt"
2019-03-25 07:58:34,487 DEBUG    Loading file "<journal folder>/2014-01.txt"
2019-03-25 07:58:34,550 DEBUG    Loading file "<journal folder>/2014-02.txt"
2019-03-25 07:58:34,624 DEBUG    Loading file "<journal folder>/2014-03.txt"
2019-03-25 07:58:34,751 DEBUG    Loading file "<journal folder>/2014-04.txt"
2019-03-25 07:58:34,842 DEBUG    Loading file "<journal folder>/2014-05.txt"
2019-03-25 07:58:34,891 DEBUG    Loading file "<journal folder>/2014-06.txt"
2019-03-25 07:58:34,903 DEBUG    Loading file "<journal folder>/2014-07.txt"
2019-03-25 07:58:34,959 DEBUG    Loading file "<journal folder>/2014-08.txt"
2019-03-25 07:58:34,986 DEBUG    Loading file "<journal folder>/2014-09.txt"
2019-03-25 07:58:35,281 DEBUG    Loading file "<journal folder>/2014-10.txt"
2019-03-25 07:58:35,553 DEBUG    Loading file "<journal folder>/2014-11.txt"
2019-03-25 07:58:35,592 DEBUG    Loading file "<journal folder>/2014-12.txt"
2019-03-25 07:58:35,627 DEBUG    Loading file "<journal folder>/2015-01.txt"
2019-03-25 07:58:35,709 DEBUG    Loading file "<journal folder>/2015-02.txt"
2019-03-25 07:58:36,289 DEBUG    Loading file "<journal folder>/2015-03.txt"
2019-03-25 07:58:36,420 DEBUG    Loading file "<journal folder>/2015-04.txt"
2019-03-25 07:58:36,976 DEBUG    Loading file "<journal folder>/2015-05.txt"
2019-03-25 07:58:37,201 DEBUG    Loading file "<journal folder>/2015-06.txt"
2019-03-25 07:58:37,279 DEBUG    Loading file "<journal folder>/2015-07.txt"
2019-03-25 07:58:37,393 DEBUG    Loading file "<journal folder>/2015-08.txt"
2019-03-25 07:58:37,588 DEBUG    Loading file "<journal folder>/2015-09.txt"
2019-03-25 07:58:37,774 DEBUG    Loading file "<journal folder>/2015-10.txt"
2019-03-25 07:58:38,287 DEBUG    Loading file "<journal folder>/2015-11.txt"
2019-03-25 07:58:38,509 DEBUG    Loading file "<journal folder>/2015-12.txt"
2019-03-25 07:58:39,524 DEBUG    Loading file "<journal folder>/2016-01.txt"
2019-03-25 07:58:39,565 DEBUG    Loading file "<journal folder>/2016-02.txt"
2019-03-25 07:58:39,592 DEBUG    Loading file "<journal folder>/2016-03.txt"
2019-03-25 07:58:39,742 DEBUG    Loading file "<journal folder>/2016-04.txt"
2019-03-25 07:58:41,193 DEBUG    Loading file "<journal folder>/2016-05.txt"
2019-03-25 07:58:41,408 DEBUG    Loading file "<journal folder>/2016-06.txt"
2019-03-25 07:58:42,823 DEBUG    Loading file "<journal folder>/2016-07.txt"
2019-03-25 07:58:43,782 DEBUG    Loading file "<journal folder>/2016-08.txt"
2019-03-25 07:58:44,311 DEBUG    Loading file "<journal folder>/2016-09.txt"
2019-03-25 07:58:44,452 DEBUG    Loading file "<journal folder>/2016-10.txt"
2019-03-25 07:58:44,801 DEBUG    Loading file "<journal folder>/2016-11.txt"
2019-03-25 07:58:45,367 DEBUG    Loading file "<journal folder>/2016-12.txt"
2019-03-25 07:58:46,031 DEBUG    Loading file "<journal folder>/2017-01.txt"
2019-03-25 07:58:46,403 DEBUG    Loading file "<journal folder>/2017-02.txt"
2019-03-25 07:58:46,482 DEBUG    Loading file "<journal folder>/2017-03.txt"
2019-03-25 07:58:46,839 DEBUG    Loading file "<journal folder>/2017-04.txt"
2019-03-25 07:58:46,974 DEBUG    Loading file "<journal folder>/2017-05.txt"
2019-03-25 07:58:47,448 DEBUG    Loading file "<journal folder>/2017-06.txt"
2019-03-25 07:58:47,689 DEBUG    Loading file "<journal folder>/2017-07.txt"
2019-03-25 07:58:48,153 DEBUG    Loading file "<journal folder>/2017-08.txt"
2019-03-25 07:58:48,457 DEBUG    Loading file "<journal folder>/2017-09.txt"
2019-03-25 07:58:48,851 DEBUG    Loading file "<journal folder>/2017-10.txt"
2019-03-25 07:58:49,144 DEBUG    Loading file "<journal folder>/2017-11.txt"
2019-03-25 07:58:49,194 DEBUG    Loading file "<journal folder>/2017-12.txt"
2019-03-25 07:58:49,214 DEBUG    Loading file "<journal folder>/2018-01.txt"
2019-03-25 07:58:49,833 DEBUG    Loading file "<journal folder>/2018-03.txt"
2019-03-25 07:58:49,876 DEBUG    Loading file "<journal folder>/2018-04.txt"
2019-03-25 07:58:50,119 DEBUG    Loading file "<journal folder>/2018-05.txt"
2019-03-25 07:58:50,183 DEBUG    Loading file "<journal folder>/2018-06.txt"
2019-03-25 07:58:50,237 DEBUG    Loading file "<journal folder>/2018-07.txt"
2019-03-25 07:58:50,319 DEBUG    Loading file "<journal folder>/2018-08.txt"
2019-03-25 07:58:50,456 DEBUG    Loading file "<journal folder>/2018-09.txt"
2019-03-25 07:58:50,599 DEBUG    Loading file "<journal folder>/2018-10.txt"
2019-03-25 07:58:50,830 DEBUG    Loading file "<journal folder>/2018-11.txt"
2019-03-25 07:58:51,008 DEBUG    Loading file "<journal folder>/2018-12.txt"
2019-03-25 07:58:51,136 DEBUG    Loading file "<journal folder>/2019-01.txt"
2019-03-25 07:58:57,234 DEBUG    Loading file "<journal folder>/2019-02.txt"
2019-03-25 07:58:57,715 DEBUG    Loading file "<journal folder>/2019-03.txt"
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-2015-12-17.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-Notebook-2016-01-06.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-Notebook-2016-01-17.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-Notebook-2016-02-07.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-Notebook-2016-02-29.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-Notebook-2016-03-31.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-labjournal-2016-05-01.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-labjournal-2016-06-08.zip is not a valid month filename
2019-03-25 07:58:57,769 DEBUG    RedNotebook-Backup-labjournal-2016-07-03.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2016-08-02.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2016-08-03.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2016-08-09.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2016-09-02.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2016-10-07.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2016-11-08.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2016-12-05.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2017-02-14.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2017-04-17.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2017-05-21.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2017-07-18.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2017-09-20.zip is not a valid month filename
2019-03-25 07:58:57,770 DEBUG    RedNotebook-Backup-labjournal-2018-01-10.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2018-04-23.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2018-06-10.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2018-07-12.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2018-10-22.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2019-01-19.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2019-01-29.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2019-02-12.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    RedNotebook-Backup-labjournal-2019-03-03.zip is not a valid month filename
2019-03-25 07:58:57,771 DEBUG    Finished loading files in dir "<journal folder>/"
2019-03-25 07:58:58,891 INFO     Connected Signals: [<Signals.SIGHUP: 1>, <Signals.SIGINT: 2>, <Signals.SIGQUIT: 3>, <Signals.SIGABRT: 6>, <Signals.SIGTERM: 15>, <Signals.SIGTSTP: 20>]
2019-03-25 07:58:58,891 DEBUG    Start took 28.13469433784485 seconds
2019-03-25 07:58:58,891 DEBUG    Trying to enter the gtk main loop
2019-03-25 07:58:58,950 DEBUG    Update the cloud
2019-03-25 07:59:01,106 DEBUG    Cloud updated
2019-03-25 07:59:01,110 INFO     Last backup was made 41 days ago
2019-03-25 07:59:01,111 WARNING  Last backup is older than 30 days.
2019-03-25 07:59:13,001 DEBUG    Main frame destroyed
2019-03-25 07:59:13,002 INFO     Nothing to save
2019-03-25 07:59:13,003 INFO     Goodbye!
2019-03-25 07:59:13,010 INFO     Peak memory: 134541192 KiB
```

**Comment by jendrikseipp** [comment]
Thanks for the report. That is strange indeed. Do you notice a similar discrepancy in speed for other applications as well, or is it only RedNotebook? Could you please report the problem to the flatpak devs (see https://github.com/flatpak/flatpak/issues/1768 for a similar issue)?

**Comment by takluyver** [comment]
I think this was meant to be closed when #2 was merged.

**Comment by jendrikseipp** [comment]
Thanks for the heads up!
